importer/yarn_to_gd.gd

```diff
@@ -78,61 +88,41 @@ static func jump(target: String):
 		"current_function = \"%s\"" % target,
 		"return %s()" % target
 	]
-
-static func options(opts: Array):
-	var options_str = "["
+	
+static func build_options(opts: Array):
+	var parsed_options = []
 	for option in opts:
-		options_str += "\"" + option.message + "\", "
+		parsed_options.append("\"" + option.message + "\".format(variables)")
```

Comment:
I wanted to allow for variable interpolation in the menu options here.

Reply:
Good point, generated code looks to be alright :D